scss/_nav.scss

```diff
@@ -105,7 +105,7 @@
     }
 
     .nav-link {
-      text-transform: uppercase;
+      text-transform: none;
       background-color: $nav-tabs-link-active-bg;
```

Comment:
Should we not just remove the text-transform?

Reply:
NVM none is better, since this will add an underline if not there, and adding a text-transform utility class will still work.